doc/dune.mld

```diff
@@ -94,21 +94,21 @@ have double underscores meaning they are hidden. Only the non-hidden module
 [Lib] is linked, and during this process, the
 the modules [A] and [B] are expanded because they are aliases of hidden
 modules. All references to [Lib__A] and [Lib__B] are replaced with the canonical
-paths [Lib.A] and [Lib.B], so in this way odoc presents the library as entirely
+paths [Lib.A] and [Lib.B], so in this way, [odoc] presents the library as entirely
 contained within the module [Lib].
 
 {2 Hand-Written Top-Level Module}
 
 In some cases it's desirable to hand-write the top-level library module. This
-is usually done because some of the modules within the library are intended to
+is usually done because some modules in the library are intended to
 be internal only and not exposed. Dune will notice that a module exists with
-the name of the library ([lib.ml] in this case), so instead it will create the
-file [lib__.ml]. The contents of this are identical to the previous section, 
+the library's name ([lib.ml] in this case), so instead it will create the
+file [lib__.ml]. Its contents are identical to the previous section, 
 with aliases for all modules. The canonical tags on the aliases are, as before,
-to [Lib.A] and [Lib.B]. These are references to module aliases that should be
+[Lib.A] and [Lib.B]. These are references to module aliases that should be
```

Comment:
`to` should likely be kept, or maybe replacing "are" with "point":

> The canonical tags on the aliases, as before, point to [Lib.A] and[Lib.b]

Reply:
Another possibility:

> The canonical tags on the aliases are, as before, references to [Lib.A] and[Lib.B].

Is that better?

Reply:
Yes! The second one is more clear. 🙂

Reply:
Its contents are identical to the previous section, 
with aliases for all modules. Like before, the canonical tags on these aliases
are references to [Lib.A] and [Lib.B]. These references should be present in [lib.ml].

It's less repetitive, but is it still accurate?

Reply:
Unfortunately, not perfectly accurate :sweat_smile: It is not the references which are present in [lib.ml], but the target of the references...

I believe we are close to something good and accurate! I think this is correct:
> ```
> Its contents are identical to the previous section, with aliases for all modules. Like before, the canonical tags on these aliases are references to [Lib.A] and [Lib.B]. These module should be present in [lib.ml] as module aliases.
> ```

Reply:
See if li 108 reads better now.